lib/logstash/inputs/http.rb

```diff
@@ -142,10 +143,12 @@ def run(queue)
           event.set("headers", req)
           decorate(event)
           queue << event
+          metric.increment(:events)
```

Comment:
I think theses metrics are or should be collected at a higher level?
What is the purpose of this one?